pogoiv/iv_calculator.py

```diff
@@ -77,13 +77,45 @@ def get_ivs(self, pokemon_name, current_cp, current_health, dust_to_upgrade, pow
 
         return possible_stats
 
-    def get_ivs_across_powerups(self, pokemon_name, powerup_stats):
+    def appraisal_filter(self, response, appraisal):
+        filtered_response = []
+        """ Filters the resulting iv set according to the appraisal values. """
+        """ Global appraisal ranges. """
```

Comment:
For inline comments, we try to roughly follow pep8 (not a bad idea in general for Python PRs 😉.

https://www.python.org/dev/peps/pep-0008/#inline-comments